fusion/swift/YelpAPISample/main.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  main.swift
+//  YelpAPISample
+//
+
+import YelpAPI
+import BrightFutures
+
+// Fill in your app keys here from
+// https://www.yelp.com/developers/v3/manage_app
+let appId = ""
```

Comment:
`appId` looks great.

Should we do `appSecret` to match https://www.yelp.com/developers/v3/manage_app and add some namespacing?